diff --git a/node_modules/@ethersproject/networks/lib.esm/index.js b/node_modules/@ethersproject/networks/lib.esm/index.js
index ac09121..88962fa 100644
--- a/node_modules/@ethersproject/networks/lib.esm/index.js
+++ b/node_modules/@ethersproject/networks/lib.esm/index.js
@@ -131,6 +131,18 @@ const networks = {
         _defaultProvider: ethDefaultProvider("goerli")
     },
     kintsugi: { chainId: 1337702, name: "kintsugi" },
+    ewc: {
+        chainId: 246,
+        ensAddress: "0x0A6d64413c07E10E890220BBE1c49170080C6Ca0",
+        name: "ewc",
+        _defaultProvider: etcDefaultProvider("https://rpc.energyweb.org", "ewc"),
+    },
+    volta: {
+        chainId: 73799,
+        ensAddress: "0xd7CeF70Ba7efc2035256d828d5287e2D285CD1ac",
+        name: "volta",
+        _defaultProvider: etcDefaultProvider("https://volta-rpc.energyweb.org", "volta"),
+    },
     // ETC (See: #351)
     classic: {
         chainId: 61,
diff --git a/node_modules/@ethersproject/networks/lib/index.js b/node_modules/@ethersproject/networks/lib/index.js
index bf5c557..c9b5022 100644
--- a/node_modules/@ethersproject/networks/lib/index.js
+++ b/node_modules/@ethersproject/networks/lib/index.js
@@ -133,6 +133,18 @@ var networks = {
         _defaultProvider: ethDefaultProvider("goerli")
     },
     kintsugi: { chainId: 1337702, name: "kintsugi" },
+    ewc: {
+        chainId: 246,
+        ensAddress: "0x0A6d64413c07E10E890220BBE1c49170080C6Ca0",
+        name: "ewc",
+        _defaultProvider: etcDefaultProvider("https://rpc.energyweb.org", "ewc"),
+    },
+    volta: {
+        chainId: 73799,
+        ensAddress: "0xd7CeF70Ba7efc2035256d828d5287e2D285CD1ac",
+        name: "volta",
+        _defaultProvider: etcDefaultProvider("https://volta-rpc.energyweb.org", "volta"),
+    },
     // ETC (See: #351)
     classic: {
         chainId: 61,
diff --git a/node_modules/@ethersproject/networks/src.ts/index.ts b/node_modules/@ethersproject/networks/src.ts/index.ts
index 9affd43..d306cc4 100644
--- a/node_modules/@ethersproject/networks/src.ts/index.ts
+++ b/node_modules/@ethersproject/networks/src.ts/index.ts
@@ -162,6 +162,18 @@ const networks: { [name: string]: Network } = {
 
     kintsugi: { chainId: 1337702, name: "kintsugi" },
 
+    ewc: {
+        chainId: 246,
+        ensAddress: "0x0A6d64413c07E10E890220BBE1c49170080C6Ca0",
+        name: "ewc",
+        _defaultProvider: etcDefaultProvider("https://rpc.energyweb.org", "ewc"),
+    },
+    volta: {
+        chainId: 73799,
+        ensAddress: "0xd7CeF70Ba7efc2035256d828d5287e2D285CD1ac",
+        name: "volta",
+        _defaultProvider: etcDefaultProvider("https://volta-rpc.energyweb.org", "volta"),
+    },
 
     // ETC (See: #351)
     classic: {
