diff --git a/node_modules/@usedapp/core/dist/esm/src/constants/chainId.d.ts b/node_modules/@usedapp/core/dist/esm/src/constants/chainId.d.ts
index 8862113..65a3ad0 100644
--- a/node_modules/@usedapp/core/dist/esm/src/constants/chainId.d.ts
+++ b/node_modules/@usedapp/core/dist/esm/src/constants/chainId.d.ts
@@ -31,6 +31,7 @@ export declare enum ChainId {
     OptimismKovan = 69,
     Optimism = 10,
     Arbitrum = 42161,
-    ArbitrumRinkeby = 421611
+    ArbitrumRinkeby = 421611,
+    Volta = 73799,
 }
 //# sourceMappingURL=chainId.d.ts.map
\ No newline at end of file
diff --git a/node_modules/@usedapp/core/dist/esm/src/constants/chainId.js b/node_modules/@usedapp/core/dist/esm/src/constants/chainId.js
index 7556905..a199a08 100644
--- a/node_modules/@usedapp/core/dist/esm/src/constants/chainId.js
+++ b/node_modules/@usedapp/core/dist/esm/src/constants/chainId.js
@@ -1,5 +1,5 @@
 // organize imports in alphabet order to sparse the conflict when adding a new chain
-import { Localhost, Hardhat, Arbitrum, ArbitrumRinkeby, Avalanche, BSC, BSCTestnet, Cronos, CronosTestnet, Fantom, Harmony, Mainnet, Ropsten, Rinkeby, Gnosis, Goerli, Kovan, Stardust, Moonriver, MoonbaseAlpha, Moonbeam, Palm, PalmTestnet, Polygon, Mumbai, OasisEmerald, Songbird, Theta, ThetaTestnet, OptimismKovan, Optimism, } from '../model';
+import { Localhost, Hardhat, Arbitrum, ArbitrumRinkeby, Avalanche, BSC, BSCTestnet, Cronos, CronosTestnet, Fantom, Harmony, Mainnet, Ropsten, Rinkeby, Gnosis, Goerli, Kovan, Stardust, Moonriver, MoonbaseAlpha, Moonbeam, Palm, PalmTestnet, Polygon, Mumbai, OasisEmerald, Songbird, Theta, ThetaTestnet, OptimismKovan, Optimism, Volta } from '../model';
 // rough alphabet order (put network from the same chain together)
 export const DEFAULT_SUPPORTED_CHAINS = [
     Localhost,
@@ -33,6 +33,7 @@ export const DEFAULT_SUPPORTED_CHAINS = [
     ThetaTestnet,
     OptimismKovan,
     Optimism,
+    Volta,
 ];
 export var ChainId;
 (function (ChainId) {
@@ -68,5 +69,6 @@ export var ChainId;
     ChainId[ChainId["Optimism"] = 10] = "Optimism";
     ChainId[ChainId["Arbitrum"] = 42161] = "Arbitrum";
     ChainId[ChainId["ArbitrumRinkeby"] = 421611] = "ArbitrumRinkeby";
+    ChainId[ChainId["Volta"] = 73799] = "Volta";
 })(ChainId || (ChainId = {}));
 //# sourceMappingURL=chainId.js.map
\ No newline at end of file
diff --git a/node_modules/@usedapp/core/dist/esm/src/model/chain/index.d.ts b/node_modules/@usedapp/core/dist/esm/src/model/chain/index.d.ts
index 286393a..c888246 100644
--- a/node_modules/@usedapp/core/dist/esm/src/model/chain/index.d.ts
+++ b/node_modules/@usedapp/core/dist/esm/src/model/chain/index.d.ts
@@ -16,4 +16,5 @@ export * from './songbird';
 export * from './theta';
 export * from './xdai';
 export * from './optimism';
+export * from './volta';
 //# sourceMappingURL=index.d.ts.map
\ No newline at end of file
diff --git a/node_modules/@usedapp/core/dist/esm/src/model/chain/index.js b/node_modules/@usedapp/core/dist/esm/src/model/chain/index.js
index 16f7502..e13afb1 100644
--- a/node_modules/@usedapp/core/dist/esm/src/model/chain/index.js
+++ b/node_modules/@usedapp/core/dist/esm/src/model/chain/index.js
@@ -17,4 +17,5 @@ export * from './songbird';
 export * from './theta';
 export * from './xdai';
 export * from './optimism';
+export * from './volta';
 //# sourceMappingURL=index.js.map
\ No newline at end of file
diff --git a/node_modules/@usedapp/core/dist/esm/src/model/chain/volta.d.ts b/node_modules/@usedapp/core/dist/esm/src/model/chain/volta.d.ts
new file mode 100644
index 0000000..71bfb60
--- /dev/null
+++ b/node_modules/@usedapp/core/dist/esm/src/model/chain/volta.d.ts
@@ -0,0 +1,7 @@
+import { Chain } from '../../constants';
+export declare const Volta: Chain;
+declare const _default: {
+    Volta: Chain;
+};
+export default _default;
+//# sourceMappingURL=volta.d.ts.map
\ No newline at end of file
diff --git a/node_modules/@usedapp/core/dist/esm/src/model/chain/volta.js b/node_modules/@usedapp/core/dist/esm/src/model/chain/volta.js
new file mode 100644
index 0000000..d5feec2
--- /dev/null
+++ b/node_modules/@usedapp/core/dist/esm/src/model/chain/volta.js
@@ -0,0 +1,13 @@
+export const Volta = {
+    chainId: 73799,
+    chainName: "Volta",
+    isTestChain: true,
+    isLocalChain: false,
+    multicallAddress: "0xEe2ADcB5BEAa7E08AF1F1a4Ee0EFfE0DfDDEdD27",
+    getExplorerAddressLink: (address) => `${voltaExplorerUrl}/address/${address}`,
+    getExplorerTransactionLink: (transactionHash) => `${voltaExplorerUrl}/tx/${transactionHash}`,
+};
+export default {
+    Volta,
+};
+//# sourceMappingURL=volta.js.map
\ No newline at end of file
